controllers/controllers.go

```diff
@@ -118,6 +119,50 @@ func (r *instanceReconciler) instanceFromNode(node *core.Node) (*instance.Info,
 	return instance.NewInfo(addr, username, "", false, node)
 }
 
+// updateKubeletCA updates the kubelet CA in the node, by copying the kubelet CA file content to the Windows instance
+func (r *instanceReconciler) updateKubeletCA(node core.Node, contents []byte) error {
+	winInstance, err := r.instanceFromNode(&node)
+	if err != nil {
+		return errors.Wrapf(err, "error creating instance for node %s", node.Name)
+	}
+	nodeConfig, err := nodeconfig.NewNodeConfig(r.k8sclientset, r.clusterServiceCIDR, r.vxlanPort, winInstance,
+		r.signer, nil, nil, r.platform, r.dockerRuntime)
+	if err != nil {
+		return errors.Wrapf(err, "error creating nodeConfig for instance %s", winInstance.Address)
+	}
+	r.log.Info("updating kubelet CA client certificates in", "node", node.Name)
+	return nodeConfig.Windows.UpdateKubeletClientCA(contents)
```

Comment:
I'm on the fence about the instanceReconciler directly calling a `Windows` function
but its probably fine :)

Reply:
Can you add more color to that?

Reply:
See https://github.com/openshift/windows-machine-config-operator/pull/1013#discussion_r852109570
I put more detail in that comment, and thinking about it more i think the change is necessary